test/internal/eq.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var assert = require('assert');
+
+var equals = require('./equals');
+var toString = require('./toString');
```

Comment:
Because I just copied and pasted this bit to get the `of` test working for the moment. ;)